packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -199,6 +265,124 @@ function renderWithFalsyKey(props = {}) {
+];
+
+let DynamicTabs = (props = {}) => {
+
+  let [tabs, setTabs] = React.useState(items);
+  let addTab = () => {
+    let newTabs = [...tabs];
+    newTabs.push({
+      name: `Tab ${tabs.length + 1}`,
+      children: `Tab Body ${tabs.length + 1}`
+    });
+
+    setTabs(newTabs);
+  };
+
+  let removeTab = () => {
+    let newTabs = [...tabs];
+    newTabs.pop();
+    setTabs(newTabs);
+  };
+
+  return (
+    <div style={{width: '80%'}}>
+      <Tabs {...props} aria-label="Tab example" items={tabs} onSelectionChange={action('onSelectionChange')}>
+        {item => (
+          <Item
+            key={item.name}
+            textValue={item.name}
+            title={
+              <>
+                {item.icon}
+                <Text>{item.name}</Text>
+              </>
+            }>
+            <Content margin="size-160">
+              <Heading>{item.children}</Heading>
+              <Text>
+                Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+                Id cupidatat aute id ut excepteur exercitation magna pariatur. Mollit irure irure reprehenderit pariatur eiusmod proident Lorem deserunt duis cillum mollit. Do reprehenderit sit cupidatat quis laborum in do culpa nisi ipsum. Velit aliquip commodo ea ipsum incididunt culpa nostrud deserunt incididunt exercitation. In quis proident sit ad dolore tempor. Eiusmod pariatur quis commodo labore cupidatat cillum enim eiusmod voluptate laborum culpa. Laborum cupidatat incididunt velit voluptate incididunt occaecat quis do.
+                Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
+              </Text>
+            </Content>
+          </Item>
+        )}
+      </Tabs>
+      <ButtonGroup marginEnd="30px">
+        <Button variant="secondary" onPress={() => addTab()}>
+          <Text>Add Tab</Text>
+        </Button>
+        <Button variant="secondary" onPress={() => removeTab()}>
+          <Text>Remove Tab</Text>
+        </Button>
+      </ButtonGroup>
+    </div>
+  );
+};
+
+let OrientationFlip = (props = {}) => {
+  let [flipOrientation, setFlipOrientation] = React.useState(true);
+
+  return (
+    <div style={{width: '80%'}}>
+      <Tabs {...props} aria-label="Tab example" items={items} onSelectionChange={action('onSelectionChange')} orientation={flipOrientation ? 'horizontal' : 'vertical'}>
+        {item => (
+          <Item
+            key={item.name}
+            textValue={item.name}
+            title={
+              <>
+                {item.icon}
+                <Text>{item.name}</Text>
+              </>
+            }>
+            <Content margin="size-160">
```

Comment:
This Content w/ margin is to provided margins around the tab panel content.
Without:
![image](https://user-images.githubusercontent.com/9661127/99299381-01c91b00-2800-11eb-9081-910c5a157b7d.png)

With:
![image](https://user-images.githubusercontent.com/9661127/99299407-08f02900-2800-11eb-8ff9-a41425b889d8.png)

I assume the original contributer added these since it looks better, XD files don't have any examples on what the tab panel content should look like. IMO we should keep as is since it allows users to customize what the tab panel content should look like in terms of positioning relative to the tab list